pinc/Pool.inc

```diff
@@ -40,8 +40,7 @@ class Pool extends Stage
         $access_change_callback,
         $description,
         $document,
-        $project_checkedout_state,
-        $project_available_state,
+        $ordinal,
```

Comment:
If we want to decouple the activities from the project states as far as possible, we don't want to define the names of the project states in stages.inc. The way the rounds work (defining the project states with a round and a stete component) seems quite reasonable but we don't have that option for pools because the names used now are not related to the ids. One of the earlier attempts I made just used the Pool id in a switch statement  to select names for the project states. I'll try this again.

Reply:
I think SR will always have to be a different sort of thing to the PP and PPV pools where the project state can be `available` or `checked out`.  But it seems like it could have a project state of its own and move between PP and SR.

Reply:
A project that is available for PP should never be in SR.

Reply:
Quite so, what I meant was that SR cannot be like the PP and PPV pools because it can only be `available` since many people can SR  at the same time. But whereas now it is 'inside' `PP checked out` it could be a distinct project state.